components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/par/OAuth2ParEndpoint.java

```diff
@@ -232,6 +233,7 @@ private void validateAuthzRequest(HttpServletRequest request) throws ParCoreExce
 
         try {
             OAuthAuthzRequest oAuthAuthzRequest = getOAuthAuthzRequest(request);
+            setIsFAPIConformantApp(request, oAuthAuthzRequest.getClientId());
```

Comment:
check whether we can move this after req obj validation or before building oauth request